working/augmentation-libraries/feature-specification.md

```diff
@@ -180,14 +202,26 @@ declaration ::=
   | 'augment'? constructorSignature (redirection | initializers)? ';'
 ```
 
-*Note that the grammar for putting `augment` before an extension type
-declaration doesn't allow also specifying a representation field. This is by
-design. An extension type augmentation always inherits the representation field
-of the introductory declaration and can't specify it.*
+As with top-level declarations, we also reuse the abstract member syntax with a
+`static` modifier to allow declaring incomplete static fields, methods, getters,
+setters, and operators. *Example:*
+
+```dart
+class C {
+  static abstract int x;   // Incomplete static variable.
```

Comment:
Incomplete static getter and setter, really.

It's a variable declaration, syntactically, but semantically it's a getter and setter signature declaration.
(I guess it's a question of which abstraction level we're talking at. As declarations, the comments are correct, but the phrasing is "declaring incomplete static fields, methods ...", and a declaration *is* syntax, but it *declares* sementics, and there is no semantic abstract variable.)

Reply:
Well, we are in the "Syntax" section... :)

Added a parenthetical.